### ISSUE TITLE
Behaviour of two-nodes function when passing calculated_ce=True

### ISSUE THREAD

**Issue by MaximeBoulinguezPimentLab** [CenterForTheBuiltEnvironment/pythermalcomfort#41]
**Describe the bug**

For two-nodes calculation with calculate_ce=True, it return the same _set value either with "standing" or "sitting" body position.

**To Reproduce**
two_nodes(
    tdb=27,
    tr=27,
    v=1.5,
    rh=70,
    met=1.2,
    clo=0.7,
    wme=0,
    body_surface_area=1.8258,
    p_atmospheric=101325,
    body_position="sitting",
    max_skin_blood_flow=90, 
    max_sweating=500,
    round= False,
    calculate_ce = True)   
RETURNS  {'_set': 24.637554908190932}
two_nodes(
    tdb=27,
    tr=27,
    v=1.5,
    rh=70,
    met=1.2,
    clo=0.7,
    wme=0,
    body_surface_area=1.8258,
    p_atmospheric=101325,
    body_position="sitting",
    max_skin_blood_flow=90, 
    max_sweating=500,
    round= False,
    calculate_ce = False)
RETURNS _set': array(25.17981798)
two_nodes(
    tdb=27,
    tr=27,
    v=1.5,
    rh=70,
    met=1.2,
    clo=0.7,
    wme=0,
    body_surface_area=1.8258,
    p_atmospheric=101325,
    body_position="standing",
    max_skin_blood_flow=90, 
    max_sweating=500,
    round= False,
    calculate_ce = True)
RETURNS {'_set': 24.637554908190932}
two_nodes(
    tdb=27,
    tr=27,
    v=1.5,
    rh=70,
    met=1.2,
    clo=0.7,
    wme=0,
    body_surface_area=1.8258,
    p_atmospheric=101325,
    body_position="standing",
    max_skin_blood_flow=90, 
    max_sweating=500,
    round= False,
    calculate_ce = False)
RETURNS _set': array(25.29078849)

**Expected behavior**
two-nodes (.,      ,   calculate_ce=True, body_position='sitting',             )    RETURNS {'_set': 24.637554908190932}  - shouldn't it be 24.495357746584208
two-nodes (.,      ,   calculate_ce=True, body_position='standing',             )   RETURNS {'_set': 24.637554908190932} - OK
two-nodes (.,      ,   calculate_ce=False, body_position='sitting',             )   RETURNS _set': array(25.17981798)  -OK 
two-nodes (.,      ,   calculate_ce=False, body_position='standing',             ) RETURNS _set': array(25.29078849)  - OK

**Expected behavior**

All those four calculation should return four different _set  values but when calculate_ce=True it returns the same _set

**Screenshots**

This might comes from this call in models.py>two_nodes (lines 2218-2231)

if kwargs["calculate_ce"]:
        result = two_nodes_optimized_return_set(
            tdb,
            tr,
            v,
            met,
            clo,
            vapor_pressure,
            wme,
            body_surface_area,
            p_atmospheric,
            1,
        )
        return {"_set": result}

The ""1"" after **p_atmospheric** stands for the **body_position** in the **_two_nodes_optimized_return_set_** function , so when passing **calculte_ce=True** it will always pass the **body_position='1'** to the **two_nodes_optimized_return_set**.

Because the test is made on **body_position == 'sitting'**  later in **two_nodes_optimized** (line 113), it will always return the _set value for a 'standing' person even if the input is 'sitting'

I tried to find an explanation in the papers you provided, expecting a limitation on the cooling effect calculation. I was expecting a comment stating that it cannot be performed on a sitting person, but I did not find anything.

Passing calculate_ce to True only returns the SET, i would be interesting to also have all other outputs from the two-node model as a complete dict.

I would appreciate if you can help me better understand this function's behaviour.

Regards, 
Maxime

**Desktop (please complete the following information):**
 - OS: win 10
 - VSCODE
 - Version : last version of pythermalcomfort 2.5.4 but I can see same args on the github version

**Comment by FedericoTartarini** [comment]
Hi, you are right and you have identified the cause of the issue. Please note that you should not really use that function with calculate_CE = True unless you are using the PMV ASHRAE model.

If you want to get all the results of the two_node model please use the [two_nodes](https://pythermalcomfort.readthedocs.io/en/latest/reference/pythermalcomfort.html#gagge-et-al-two-node-model) function directly.

May I ask you why you are setting calculate_ce = True?

**Comment by cpede** [comment]
Hi, I think that ASHRAE-55 2020 states that the SET should be calculated using the same CLO and MET as the PMV. The CE should then use the same "sitting" or "standing" as the SET.
But, that said why is the _pythermalcomfort_ using "sitting" or "standing" to calculate the clothing emissivity, ASHRAE-55 2020 simply uses this `CHR = 4.0 * SBC * Math.pow(((TCL + TR)/2.0 + 273.15), 3.0) * 0.72;` as described in the standard?
-cpede

**Comment by FedericoTartarini** [comment]
>Hi, I think that ASHRAE-55 2020 states that the SET should be calculated using the same CLO and MET as the PMV. The CE should then use the same "sitting" or "standing" as the SET.

The ASHRAE standard contained an error and we have changed the definition of the SET since was not the same as the one proposed by Gagge in his original work. See [ASHRAE 55 Addendum G](https://www.ashrae.org/file%20library/technical%20resources/standards%20and%20guidelines/standards%20addenda/55_2020_g_20220429.pdf)

>But, that said why is the pythermalcomfort using "sitting" or "standing" to calculate the clothing emissivity, ASHRAE-55 2020 simply uses this CHR = 4.0 * SBC * Math.pow(((TCL + TR)/2.0 + 273.15), 3.0) * 0.72; as described in the standard?

This is another issue with the old version of the Standard, which I just realised I may have never flagged to the person chairing the ASHRAE 55 committee. The code in the ASHRAE 55 is not including the correction for a person that is sitting or standing, instead, it is using an average value. Thank you for pointing this out.
